-------------------------------------------------------------------------------
Test set: org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator
-------------------------------------------------------------------------------
Tests run: 11, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 13.461 sec <<< FAILURE!
testAuthenticationHttpClientPostWithMultiAuthHandler(org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator)  Time elapsed: 1.523 sec  <<< FAILURE!
java.lang.AssertionError: expected:<200> but was:<401>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at org.apache.hadoop.security.authentication.client.AuthenticatorTestCase.doHttpClientRequest(AuthenticatorTestCase.java:236)
	at org.apache.hadoop.security.authentication.client.AuthenticatorTestCase._testAuthenticationHttpClient(AuthenticatorTestCase.java:246)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator$6.call(TestKerberosAuthenticator.java:219)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator$6.call(TestKerberosAuthenticator.java:216)
	at org.apache.hadoop.security.authentication.KerberosTestUtils$1.run(KerberosTestUtils.java:120)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.authentication.KerberosTestUtils.doAs(KerberosTestUtils.java:117)
	at org.apache.hadoop.security.authentication.KerberosTestUtils.doAsClient(KerberosTestUtils.java:133)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator.testAuthenticationHttpClientPostWithMultiAuthHandler(TestKerberosAuthenticator.java:216)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

testAuthenticationHttpClient(org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator)  Time elapsed: 1.109 sec  <<< FAILURE!
java.lang.AssertionError: expected:<200> but was:<401>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at org.apache.hadoop.security.authentication.client.AuthenticatorTestCase.doHttpClientRequest(AuthenticatorTestCase.java:236)
	at org.apache.hadoop.security.authentication.client.AuthenticatorTestCase._testAuthenticationHttpClient(AuthenticatorTestCase.java:246)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator$3.call(TestKerberosAuthenticator.java:157)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator$3.call(TestKerberosAuthenticator.java:154)
	at org.apache.hadoop.security.authentication.KerberosTestUtils$1.run(KerberosTestUtils.java:120)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.authentication.KerberosTestUtils.doAs(KerberosTestUtils.java:117)
	at org.apache.hadoop.security.authentication.KerberosTestUtils.doAsClient(KerberosTestUtils.java:133)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator.testAuthenticationHttpClient(TestKerberosAuthenticator.java:154)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

testAuthenticationHttpClientPost(org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator)  Time elapsed: 1.122 sec  <<< FAILURE!
java.lang.AssertionError: expected:<200> but was:<401>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at org.apache.hadoop.security.authentication.client.AuthenticatorTestCase.doHttpClientRequest(AuthenticatorTestCase.java:236)
	at org.apache.hadoop.security.authentication.client.AuthenticatorTestCase._testAuthenticationHttpClient(AuthenticatorTestCase.java:246)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator$4.call(TestKerberosAuthenticator.java:171)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator$4.call(TestKerberosAuthenticator.java:168)
	at org.apache.hadoop.security.authentication.KerberosTestUtils$1.run(KerberosTestUtils.java:120)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.authentication.KerberosTestUtils.doAs(KerberosTestUtils.java:117)
	at org.apache.hadoop.security.authentication.KerberosTestUtils.doAsClient(KerberosTestUtils.java:133)
	at org.apache.hadoop.security.authentication.client.TestKerberosAuthenticator.testAuthenticationHttpClientPost(TestKerberosAuthenticator.java:168)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

